solutions/standard/variables.tf

```diff
@@ -0,0 +1,202 @@
+  default     = "standard"
+}
+
+variable "root_ca_name" {
+  type        = string
+  description = "Name of the Root CA to create for a private_cert secret engine. Only used when var.existing_sm_instance_guid is false"
+  default     = "root-ca"
+}
+
+variable "intermediate_ca_name" {
+  type        = string
+  description = "Name of the Intermediate CA to create for a private_cert secret engine. Only used when var.existing_sm_instance_guid is false"
+  default     = "intermediate-ca"
+}
+
+variable "certificate_template_name" {
+  type        = string
+  description = "Name of the Certificate Template to create for a private_cert secret engine. When var.existing_sm_instance_guid is true, then it has to be the existing template name that exists in the private cert engine."
+  default     = "my-template"
+}
+
+variable "create_policy" {
+  description = "Set to true to create a new access group (using the value of var.access_group_name) with a VPN Client role"
+  type        = bool
+  default     = true
+}
+
+variable "vpn_client_access_group_users" {
+  description = "List of users in the Client to Site VPN Access Group"
+  type        = list(string)
+  default     = []
+}
+
+variable "access_group_name" {
+  type        = string
+  description = "Name of the IAM Access Group to create if var.create_policy is true"
+  default     = "client-to-site-vpn-access-group"
+}
+
+variable "vpn_server_routes" {
+  type = map(object({
+    destination = string
+    action      = string
+  }))
+  description = "Map of server routes to be added to created VPN server."
+  default = {
+    "vpc-10" = {
+      destination = "10.0.0.0/8"
+      action      = "translate"
+    }
+    "route-vpn-2-services" = {
+      destination = "166.9.0.0/16"
+      action      = "translate"
+    }
+    "route-vpn-2-dns" = {
+      destination = "161.26.0.0/24"
+      action      = "translate"
+    }
+  }
+}
+
+variable "root_ca_max_ttl" {
+  type        = string
+  description = "Maximum TTL value for the root CA"
+  default     = "8760h"
+}
+
+variable "root_ca_common_name" {
+  type        = string
+  description = "Fully qualified domain name or host domain name for the certificate to be created"
+  default     = "cloud.ibm.com"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "Name of the resource group to use for this example. If not set, a resource group is created."
+  default = null
+}
+
+variable "cert_common_name" {
+  type        = string
+  description = "Fully qualified domain name or host domain name for the private certificate to be created"
+
+  validation {
+    condition     = length(var.cert_common_name) >= 4 && length(var.cert_common_name) <= 128
+    error_message = "length of cert_common_name must be >= 4 and <= 128"
+  }
+
+  validation {
+    condition     = can(regex("(.*?)", var.cert_common_name))
+    error_message = "cert_common_name must match regular expression /(.*?)/"
+  }
+}
+
+variable "port_max_in" {
```

Comment:
See my comment in quickstart about giving these variables more specific names.

Also - it seems these are already defined below with more specific names - see `port_max_webin`, `port_min_webin`, `port_max_telin`, `port_min_telin`.  Do we really need all of these definitions or can they be re-used?

Reply:
Yes..We can re-use the port variables. Updated accordingly.